### PR TITLE
Astro 1210 text input

### PR DESCRIPTION
**Pull request by Cheshire89**
https://rocketcom.atlassian.net/browse/ASTRO-1210
light styling update for text inputs

### CODE REVIEW

**Review by dmcalester — APPROVED** [review]

**Review by cskerbo — APPROVED** [review]

File: src/css/core/input.css [hunk 1]
```diff
@@ -84,12 +85,25 @@
   border-radius: 3px;
```
Comment (Design-Kat) [inline src/css/core/input.css:85]: Unless there is a separate ticket for text-areas, the border and font changes should be reflected in text area as well. I'm still verifying the border- it appears it was changed and I want to confirm as I believe it was originally designed to match the buttons, will update when I've got the confirmation.

Comment (Design-Kat) [reply src/css/core/input.css:85]: @Cheshire89 Confirmed with Ty that it actually should be 3px border radius- I will put in a change for the sketch file for this.

File: src/css/core/input.css [hunk 2]
```diff
@@ -60,12 +61,12 @@
 
   height: 2.125rem;
   width: 100%;
-  padding: 0 1rem;
+  padding: 0 0.625rem;
 
   border: 1px solid var(--inputBorderColor);
-  border-radius: 3px;
+  border-radius: 4px;
```
Comment (Design-Kat) [inline src/css/core/input.css:67]: @Cheshire89 Confirmed with Ty that it actually should be 3px border radius- I will put in a change for the sketch file for this.

File: static/css/astro.core.css [hunk 3]
```diff
@@ -2022,14 +2023,14 @@ label {
 
   height: 2.125rem;
   width: 100%;
-  padding: 0 1rem;
+  padding: 0 0.625rem;
 
   border: 1px solid var(--inputBorderColor);
-  border-radius: 3px;
+  border-radius: 4px;
```
Comment (Design-Kat) [inline static/css/astro.core.css:2029]: @Cheshire89 Confirmed with Ty that it actually should be 3px border radius- I will put in a change for the sketch file for this.

**Review by Design-Kat — CHANGES_REQUESTED** [review]
So, 2 things: 1) It seems that the sketch file has the input's border radius changed to 4px, I have confirmed with Ty that it should match the 3px border radius of buttons. 2) Any changes made to text inputs should probably matched on Text areas (I can't think of a reason why they wouldn't be, @amalyah @dmcalester could you confirm please? There's no special reason I'm not thinking of, is there?)

**Review by Design-Kat — APPROVED** [review]

**Pull request merged**